### ISSUE TITLE
Feature Request: Remember resize

### ISSUE THREAD

**Issue by Peterragheb** [outl1ne/nova-resizable#3]
Can you please make it that the column remembers its last resize width?

**Comment by keizah7** [comment]
It would be great

**Comment by eimantaaas** [comment]
+1

**Comment by sfinktah** [comment]
@Peterragheb I'm not sure sure that would necessarily desirable, e.g. if the table is consequently viewed at a different viewport size.  

it would also introduce technical issues, as the individual columns are automatically resized so the total of column widths do not fall below the table width.

**Comment by sfinktah** [comment]
Update: I implemented minColumnWidth and maxColumnWidth, it wasn't actually that hard.  I smuggled the info in via `meta` and wrote it to `data-min-column-width` attributes in the headers... from there, a few quick hacks of the resizing javascript had it reading in the values.

Since I also took the opportunity to write the name of the resource into the `table` tag, it would be a simple task to use LocalStorage to write unique keys `resource`_`field`_`width` and thus remember the column size.  It could probably even be done without any of the modifications I made.
